coverage/coverage-summary.json

```diff
@@ -0,0 +1,13 @@
+{"total": {"lines":{"total":634,"covered":523,"skipped":0,"pct":82.49},"statements":{"total":668,"covered":551,"skipped":0,"pct":82.49},"functions":{"total":185,"covered":149,"skipped":0,"pct":80.54},"branches":{"total":298,"covered":227,"skipped":0,"pct":76.17}}
+,"C:\\Users\\danie\\Documents\\workspace\\Artemis\\mobx-async-store\\spec\\ExampleApp.js": {"lines":{"total":22,"covered":22,"skipped":0,"pct":100},"functions":{"total":9,"covered":9,"skipped":0,"pct":100},"statements":{"total":22,"covered":22,"skipped":0,"pct":100},"branches":{"total":0,"covered":0,"skipped":0,"pct":100}}
```

Comment:
this explicit path to your user workspace makes me think something is not correctly set up

Reply:
perhaps we should be ignoring this files, and everything else in `coverage` except for the svg badges?

Reply:
Yup, that's it, gotta ignore everything but the badges